client/src/AddFriend.js

```diff
@@ -112,9 +113,7 @@ class AddFriend extends React.Component {
           {this.state.isLoading ? loadButton() : submitButton()}
         </form>
 
-        <div>
-          {displayAlert(this.state)}
-        </div>
+        <Alerts state={this.state} />
```

Comment:
You're passing a prop down called `state` which is kind of confusing -- it's better to be explicit about what fields on `state` you want to pass down.

Reply:
eg `statusCode` or even better `alertType`